src/components/Popups/PopupReply.tsx

```diff
@@ -1,42 +1,27 @@
 import React from 'react';
-import Button from 'box-ui-elements/es/components/button';
-import PrimaryButton from 'box-ui-elements/es/components/primary-button';
 import { KEYS } from 'box-ui-elements/es/constants';
-import { FormattedMessage } from 'react-intl';
-import messages from './messages';
+import ReplyForm from '../ReplyForm';
 import PopupBase from './PopupBase';
-import './PopupReply.scss';
 
 export type Props = {
     className?: string;
     defaultValue?: string;
-    onCancel: (text?: string) => void;
-    onChange: (text?: string) => void;
+    onCancel: (text: string) => void;
+    onChange: (text: string) => void;
     onSubmit: (text: string) => void;
     reference: HTMLElement;
 };
 
-export default function PopupReply({ defaultValue, onCancel, onChange, onSubmit, ...rest }: Props): JSX.Element {
-    const [text, setText] = React.useState('');
+export default function PopupReply({ onCancel, onChange, onSubmit, ...rest }: Props): JSX.Element {
+    let text = '';
```

Comment:
This change means we're storing the text state in three places instead of one. It seems like the textarea itself could be its own component, which would the ref to be forwarded and for state to be managed by the store.